src/sdg/producten/tests/test_update_catalogs.py

```diff
@@ -1,27 +1,286 @@
+import datetime
+from datetime import date
+
+from django.core.management import call_command
 from django.test import TestCase
 
+from sdg.core.models import ProductenCatalogus
+from sdg.core.tests.factories.catalogus import ProductenCatalogusFactory
+from sdg.organisaties.tests.factories.overheid import LokaleOverheidFactory
+from sdg.producten.models.localized import LocalizedProduct
+from sdg.producten.models.product import Product, ProductVersie
+from sdg.producten.tests.factories.localized import LocalizedProductFactory
+from sdg.producten.tests.factories.product import (
+    ProductVersieFactory,
+    ReferentieProductFactory,
+)
+
 
 class UpdateCatalogTests(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.producten_catalogus_database = ProductenCatalogus.objects.all()
+        self.producten_database = Product.objects.all()
+        self.producten_versies_database = ProductVersie.objects.all()
+        self.localized_producten_database = LocalizedProduct.objects.all()
+
     def test_create_catalog_from_reference_if_enabled(self):
-        pass
+        ref_overheid = LokaleOverheidFactory.create(
+            automatisch_catalogus_aanmaken=False
+        )
+        ProductenCatalogusFactory.create(
+            lokale_overheid=ref_overheid,
+            is_referentie_catalogus=True,
+        )
+
+        LokaleOverheidFactory.create(
+            automatisch_catalogus_aanmaken=True,
+            organisatie__owms_end_date=None,
+        )
+
+        self.assertEqual(self.producten_catalogus_database.count(), 1)
+        call_command("update_catalogs")
+        self.assertNotEqual(self.producten_catalogus_database.count(), 1)
```

Comment:
So, what did you test here? The command should create a catalog but after the command there's still only 1?

Reply:
I test if the catalogs do not equal 1, but i'm gonna change this to test if we got 2 product catalogus

Reply:
I test if the catalogs do not equal 1, but i'm gonna change this to test if we got 2 product catalogus